File: 프로그래머스/lv0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        my_string = my_string.replaceAll("[a-z|A-Z]","");
        String[] answers = my_string.split("");
        for(int i=0;i<answers.length;i++){
            answer+=Integer.parseInt(answers[i]);
        }
        return answer;
    }
}

File: 프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.js
function solution(my_string) {
    var answer = 0;
    for(let i =0;i<my_string.length;i++){
        if(!Number.isNaN(parseInt(my_string[i]))){
            answer += parseInt(my_string[i]);
        }
    }
    return answer;
}